modules/virt-features-for-storage-matrix.adoc

```diff
@@ -122,6 +127,7 @@ ifdef::virt-features-for-storage,virt-importing-rhv-vm[]
 --
```

Comment:
I think the `virt-importing-rhv-vm` ifdef is a mistake. There is no "#3" in the RHV storage matrix, so this note does not belong with RHV VM import.

Reply:
Removed virt-importing-rhv-vm from if-def. Sorry @apinnick that was a copy and paste error. Thanks for catching it.

Please verify this is fixed and we can move his to QE.

Bob